packages/calcite-components/src/components/meter/meter.stories.ts

```diff
@@ -0,0 +1,76 @@
+import { storyFilters } from "../../../.storybook/helpers";
+import readme from "./readme.md";
+import { html } from "../../../support/formatting";
+
+export default {
+  title: "Components/Meter",
+  parameters: {
+    notes: readme,
+  },
+  ...storyFilters(),
+};
+
+export const simple = (): string => html`<calcite-meter min="0" max="100" low="25" high="75" />`;
```

Comment:
Do we need closing tags here?

Reply:
Applies to other usages without closing tags and not JSX

Reply:
Updated